lambdas/process-journey-event/src/test/java/uk/gov/di/ipv/core/processjourneyevent/JourneyMapTest.java

```diff
@@ -30,15 +30,17 @@
 
 class JourneyMapTest {
 
-    @ParameterizedTest
-    @EnumSource
-    void shouldHandleSameEventsForAllCris(IpvJourneyTypes journeyType) throws IOException {
-        var stateMachineInitializer = new StateMachineInitializer(journeyType);
-        var stateMachine = stateMachineInitializer.initialize();
-
+    @Test
```

Comment:
These tests should check for consistency across all journey maps, not just within each individually